src/test/java/bio/terra/janitor/integration/common/configuration/IntegrationTestConfiguration.java

```diff
@@ -8,8 +10,8 @@
 /** Configs in integration test. */
 @Component
 @EnableConfigurationProperties
-@ConfigurationProperties(prefix = "test")
-public class TestConfiguration {
+@ConfigurationProperties(prefix = "it")
+public class IntegrationTestConfiguration {
```

Comment:
Why do we have this class and the TestConfiguration class?

Reply:
TestConfiguration is for unit test configs. And integrationTestIntegration is for integration test.